docs/sources/api/_index.md

```diff
@@ -805,13 +806,34 @@ In microservices mode, the `/flush` endpoint is exposed by the ingester.
 
 ## `POST /ingester/flush_shutdown`
 
+**Deprecated**: Please use `/ingester/shutdown?flush=true` instead.
+
 `/ingester/flush_shutdown` triggers a shutdown of the ingester and notably will _always_ flush any in memory chunks it holds.
 This is helpful for scaling down WAL-enabled ingesters where we want to ensure old WAL directories are not orphaned,
 but instead flushed to our chunk backend.
 
 In microservices mode, the `/ingester/flush_shutdown` endpoint is exposed by the ingester.
 
-### `GET /distributor/ring`
+## `POST /ingester/shutdown`
+
+`/ingester/shutdown` is similar to the [`/ingester/flush_shutdown`](#post-ingesterflush_shutdown)
+endpoint, but accepts three URL query parameters `flush`, `delete_ring_tokens`, and `terminate`.
+
+**URL query parameters:**
```

Comment:
since this is a POST, curious why we opt for url params intead of a request body?

Reply:
Good point. [We've discussed this a bit](https://github.com/grafana/loki/pull/6179#discussion_r876827482). Query params are I guess easier to write when you execute the request using `curl`.

Reply:
Also, I think we do not follow any strict, restful HTTP API design guides in our endpoint design.